clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1732,6 +1742,11 @@ scheduler_tmpl_t<RecordType, ReaderType>::next_record(output_ordinal_t output,
                 // The stop is exclusive.  0 does mean to do nothing (easiest
                 // to have an empty record to share the next-entry for a start skip
                 // or other cases).
+                VPRINT(this, 5,
+                       "next_record_cmp[%d]: for %d: at instr#=%" PRId64
```

Comment:
Wouldn't be clear to a reader what next_record_cmp is.

Reply:
Deleted this one; probably not as useful.